pkg/api/v1/schema.go

```diff
@@ -35,8 +35,15 @@ type DomainSpec struct {
 }
 
 type Memory struct {
```

Comment:
Would it make sense to model the memory part more like the Pods API?

E.g. more like: 

```yaml
    resources:
      limits:
        memory: "800Mi"
        cpu: "1"
      requests:
        memory: "600Mi"
        cpu: "0.5"
```

than

```yaml
    memory:
        request: "600Mi"
        limits: "800Mi"
```

Reply:
But we will eventually need memory.required as well (limit from the bottom)

Reply:
How about `requires`:

```yaml
    resources:
      limits:
        memory: "800Mi"
        cpu: "1"
      requests:
        memory: "600Mi"
        cpu: "0.5"
      requires:
        memory: "500Mi"
        cpu: "0.3"
```

The `requires` values would then probably be hints for ballooning, load balancers and eviction policies?